commit c505d943a0c387b868311b172a141851c7ab38c1
Author: Till Hofmann <thofmann@fedoraproject.org>
Date:   Wed Jan 23 11:22:55 2019 +0000

    Remove some CFLAGS modification
    
    We want to use the distro's CFLAGS as close as possible.

diff --git a/CMake/unix_config.cmake b/CMake/unix_config.cmake
index 50371cd66..4d8f8cc73 100644
--- a/CMake/unix_config.cmake
+++ b/CMake/unix_config.cmake
@@ -14,29 +14,10 @@ macro(os_set_flags)
     set(CMAKE_ARCHIVE_OUTPUT_DIRECTORY ${CMAKE_BINARY_DIR}/${CMAKE_BUILD_TYPE})
 
     set(CMAKE_POSITION_INDEPENDENT_CODE ON)
-    set(CMAKE_C_FLAGS   "${CMAKE_C_FLAGS}   -pedantic -D_DEFAULT_SOURCE")
-    set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -pedantic -Wno-missing-field-initializers")
+    set(CMAKE_C_FLAGS   "${CMAKE_C_FLAGS}   -D_DEFAULT_SOURCE")
+    set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -Wno-missing-field-initializers")
     set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -Wno-switch -Wno-multichar -Wsequence-point -Wformat -Wformat-security")
 
-    execute_process(COMMAND ${CMAKE_C_COMPILER} -dumpmachine OUTPUT_VARIABLE MACHINE)
-    if(${MACHINE} MATCHES "arm64-*" OR ${MACHINE} MATCHES "aarch64-*")
-        set(CMAKE_C_FLAGS   "${CMAKE_C_FLAGS}   -mstrict-align -ftree-vectorize")
-        set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -mstrict-align -ftree-vectorize")
-    elseif(${MACHINE} MATCHES "arm-*")
-        set(CMAKE_C_FLAGS   "${CMAKE_C_FLAGS}   -mfpu=neon -mfloat-abi=hard -ftree-vectorize -latomic")
-        set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -mfpu=neon -mfloat-abi=hard -ftree-vectorize -latomic")
-    elseif(${MACHINE} MATCHES "powerpc64(le)?-linux-gnu")
-        set(CMAKE_C_FLAGS   "${CMAKE_C_FLAGS}   -ftree-vectorize")
-        set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -ftree-vectorize")
-    elseif(${MACHINE} MATCHES "riscv64-*")
-        set(CMAKE_C_FLAGS   "${CMAKE_C_FLAGS}   -mstrict-align -ftree-vectorize")
-        set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -mstrict-align -ftree-vectorize")
-    else()
-        set(CMAKE_C_FLAGS   "${CMAKE_C_FLAGS}   -mssse3")
-        set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -mssse3")
-        set(LRS_TRY_USE_AVX true)
-    endif(${MACHINE} MATCHES "arm64-*" OR ${MACHINE} MATCHES "aarch64-*")
-
     if(BUILD_WITH_OPENMP)
         find_package(OpenMP REQUIRED)
         set(CMAKE_C_FLAGS "${CMAKE_C_FLAGS} ${OpenMP_C_FLAGS}")
